### PR TITLE
add method to extract some metadata from raw json

### PR DESCRIPTION
**Pull request by matthyx**

### CODE REVIEW

File: armometadata/k8sutils_test.go [hunk 1]
```diff
@@ -119,3 +121,71 @@ func TestLoadClusterConfig(t *testing.T) {
...
+			labels: map[string]string{
+				"kubescape.io/image-id":   "quay-io-kubescape-kubescape-sha256-608b85d3de51caad84a2bfe089ec",
+				"kubescape.io/image-name": "quay-io-kubescape-kubescape",
+			},
+			creationTs: "2023-11-16T10:13:40Z",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			input, err := os.ReadFile(fmt.Sprintf("testdata/%s.json", tt.name))
```
Comment (amirmalka) [inline armometadata/k8sutils_test.go:182]: @matthyx I think you forgot to commit the testdata dir?

Comment (matthyx) [reply armometadata/k8sutils_test.go:182]: sorry, done

File: armometadata/k8sutils.go [hunk 2]
```diff
@@ -102,3 +104,45 @@ func LoadConfig(configPath string) (*ClusterConfig, error) {
 	err = json.Unmarshal(res, &config)
 	return config, err
 }
+
+// ExtractMetadataFromBytes extracts metadata from the JSON bytes of a Kubernetes object
+func ExtractMetadataFromJsonBytes(input []byte) (error, map[string]string, map[string]string, string) {
```
Comment (kooomix) [inline armometadata/k8sutils.go:109]: @matthyx we need also ownerReferences and resourceVersion

**Review by kooomix — CHANGES_REQUESTED** [review]

**Review by amirmalka — APPROVED** [review]

File: armometadata/k8sutils.go (hunk 2 above)
Comment (matthyx) [reply armometadata/k8sutils.go]: done

**Review by kooomix — APPROVED** [review]

**Pull request merged**